diambra/arena/stable_baselines/make_sb_env.py

```diff
@@ -1,86 +1,94 @@
 import os
+import time
 import diambra.arena
-from .wrappers.add_obs_wrap import AdditionalObsToChannel
-from .wrappers.p2_wrap import SelfPlayVsRL, VsHum, IntegratedSelfPlay
+import gym
 
 from stable_baselines import logger
 from stable_baselines.bench import Monitor
-from stable_baselines.common.misc_util import set_global_seeds
 from stable_baselines.common.vec_env import DummyVecEnv, SubprocVecEnv
+from stable_baselines.common import set_global_seeds
 
-
-def make_sb_env(seed: int, env_settings: dict, wrappers_settings: dict={},
-                traj_rec_settings: dict={}, custom_wrappers: list=None,
-                key_to_add: list=None, p2_mode: str=None, p2_policy=None,
-                start_index: int=0, allow_early_resets: bool=True,
-                start_method: str=None, no_vec: bool=False,
-                use_subprocess: bool=False):
+# Make Stable Baselines Env function
+def make_sb_env(game_id: str, env_settings: dict={}, wrappers_settings: dict={},
```

Comment:
Same here, and if we use EnvironmentSettings as a class, we can add the docstring there and effectively reuse it here by just refereing to EnvironmentSettings

Reply:
See the previous points/comments. Will resolve this as the point is the same everywhere and if we decide to modify it, I will make sure to modify every place where it applies.